src/main/java/net/openhft/chronicle/threads/LongPauser.java

```diff
@@ -26,7 +26,18 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.locks.LockSupport;
 
+import static net.openhft.chronicle.threads.LongPauser.ToStringHelper.*;
+
+/**
+ * A {@link Pauser} that implements a pausing strategy with phases of busy waiting, yielding, and sleeping,
+ * with each phase increasing in duration up to a configured limit. It is designed for scenarios where a gradual
+ * back-off is needed from active to more passive waiting states.
+ * <p>
+ * The pausing behavior begins with busy waiting, transitions to yielding, and ultimately moves to sleeping,
+ * progressively increasing the pause time from a minimum to a specified maximum duration.
+ */
 public class LongPauser implements Pauser, TimingPauser {
+    private static final long MAX_FACTOR = Jvm.isDebug() ? 100 : 1;
```

Comment:
Sorry but I am really uncomfortable with any usages of `Jvm.isDebug()` - the idea of things behaving differently inside vs outside the debugger does not make me happy!

Reply:
I agree, this may lead to surprise / confusion. Maybe we log a warning in the situations where this would be an issue and redirect the user to setting MAX_FACTOR via system prop?

Reply:
This could be worth simplifying, though the difference in the current `develop` is higher.